src/ui/components/map/providers/mapprovider.js

```diff
@@ -90,6 +90,18 @@ export default class MapProvider {
      * @type {boolean}
      */
     this._collapsePins = config.collapsePins || false;
+
+    /**
+     * Locale of the map. Defaults to 'en'
+     * @type {string}
+     */
+    this._locale = config.locale || 'en';
```

Comment:
We should make a note to update this once the other SDK i18n changes are merged in. I think this should default to the language/locale of the bundle.

Reply:
Makes sense! Adding a TODO